Sometimes rabbitmq does not start on time in AppVeyor

It seems that rabbitmq takes a bit to start. Some tests that use it work but the firsts fail.

Full log:

```
Build started
git clone -q https://github.com/src-d/go-queue.git c:\gopath\src\gopkg.in\src-d\go-queue.v1
git fetch -q origin +refs/pull/9/merge:
git checkout -qf FETCH_HEAD
set PATH=%GOPATH%\bin;c:\go\bin;%PATH%
go version
go version go1.10 windows/amd64
go get -v -t ./...
github.com/satori/go.uuid (download)
Fetching https://gopkg.in/src-d/go-errors.v0?go-get=1
Parsing meta tags from https://gopkg.in/src-d/go-errors.v0?go-get=1 (status code 200)
get "gopkg.in/src-d/go-errors.v0": found meta tag get.metaImport{Prefix:"gopkg.in/src-d/go-errors.v0", VCS:"git", RepoRoot:"https://gopkg.in/src-d/go-errors.v0"} at https://gopkg.in/src-d/go-errors.v0?go-get=1
gopkg.in/src-d/go-errors.v0 (download)
github.com/pkg/errors (download)
Fetching https://gopkg.in/src-d/go-errors.v1?go-get=1
Parsing meta tags from https://gopkg.in/src-d/go-errors.v1?go-get=1 (status code 200)
get "gopkg.in/src-d/go-errors.v1": found meta tag get.metaImport{Prefix:"gopkg.in/src-d/go-errors.v1", VCS:"git", RepoRoot:"https://gopkg.in/src-d/go-errors.v1"} at https://gopkg.in/src-d/go-errors.v1?go-get=1
gopkg.in/src-d/go-errors.v1 (download)
Fetching https://gopkg.in/vmihailenco/msgpack.v2?go-get=1
Parsing meta tags from https://gopkg.in/vmihailenco/msgpack.v2?go-get=1 (status code 200)
get "gopkg.in/vmihailenco/msgpack.v2": found meta tag get.metaImport{Prefix:"gopkg.in/vmihailenco/msgpack.v2", VCS:"git", RepoRoot:"https://gopkg.in/vmihailenco/msgpack.v2"} at https://gopkg.in/vmihailenco/msgpack.v2?go-get=1
gopkg.in/vmihailenco/msgpack.v2 (download)
github.com/stretchr/testify (download)
github.com/jpillora/backoff (download)
github.com/kelseyhightower/envconfig (download)
github.com/streadway/amqp (download)
Fetching https://gopkg.in/src-d/go-log.v1?go-get=1
Parsing meta tags from https://gopkg.in/src-d/go-log.v1?go-get=1 (status code 200)
get "gopkg.in/src-d/go-log.v1": found meta tag get.metaImport{Prefix:"gopkg.in/src-d/go-log.v1", VCS:"git", RepoRoot:"https://gopkg.in/src-d/go-log.v1"} at https://gopkg.in/src-d/go-log.v1?go-get=1
gopkg.in/src-d/go-log.v1 (download)
github.com/sirupsen/logrus (download)
Fetching https://golang.org/x/crypto/ssh/terminal?go-get=1
Parsing meta tags from https://golang.org/x/crypto/ssh/terminal?go-get=1 (status code 200)
get "golang.org/x/crypto/ssh/terminal": found meta tag get.metaImport{Prefix:"golang.org/x/crypto", VCS:"git", RepoRoot:"https://go.googlesource.com/crypto"} at https://golang.org/x/crypto/ssh/terminal?go-get=1
get "golang.org/x/crypto/ssh/terminal": verifying non-authoritative meta tag
Fetching https://golang.org/x/crypto?go-get=1
Parsing meta tags from https://golang.org/x/crypto?go-get=1 (status code 200)
golang.org/x/crypto (download)
Fetching https://golang.org/x/sys/windows?go-get=1
Parsing meta tags from https://golang.org/x/sys/windows?go-get=1 (status code 200)
get "golang.org/x/sys/windows": found meta tag get.metaImport{Prefix:"golang.org/x/sys", VCS:"git", RepoRoot:"https://go.googlesource.com/sys"} at https://golang.org/x/sys/windows?go-get=1
get "golang.org/x/sys/windows": verifying non-authoritative meta tag
Fetching https://golang.org/x/sys?go-get=1
Parsing meta tags from https://golang.org/x/sys?go-get=1 (status code 200)
golang.org/x/sys (download)
github.com/src-d/envconfig (download)
github.com/serenize/snaker (download)
github.com/x-cray/logrus-prefixed-formatter (download)
github.com/mgutz/ansi (download)
github.com/mattn/go-colorable (download)
github.com/mattn/go-isatty (download)
github.com/pkg/errors
github.com/satori/go.uuid
gopkg.in/src-d/go-errors.v0
gopkg.in/src-d/go-errors.v1
gopkg.in/vmihailenco/msgpack.v2/codes
gopkg.in/vmihailenco/msgpack.v2
github.com/jpillora/backoff
github.com/kelseyhightower/envconfig
gopkg.in/src-d/go-queue.v1
github.com/streadway/amqp
golang.org/x/sys/windows
github.com/serenize/snaker
github.com/src-d/envconfig
golang.org/x/crypto/ssh/terminal
github.com/mattn/go-isatty
github.com/sirupsen/logrus
github.com/mattn/go-colorable
github.com/mgutz/ansi
gopkg.in/src-d/go-queue.v1/memory
github.com/x-cray/logrus-prefixed-formatter
github.com/stretchr/testify/vendor/github.com/davecgh/go-spew/spew
gopkg.in/src-d/go-log.v1
github.com/stretchr/testify/vendor/github.com/pmezard/go-difflib/difflib
gopkg.in/src-d/go-queue.v1/amqp
github.com/stretchr/testify/assert
github.com/stretchr/testify/require
github.com/stretchr/testify/suite
gopkg.in/src-d/go-queue.v1/test
set PATH=C:\Program Files\erl9.2\bin;%PATH%
choco install rabbitmq --ignoredependencies -y
Chocolatey v0.10.8
Installing the following packages:
rabbitmq
By installing you accept licenses for the packages.
Progress: Downloading rabbitmq 3.7.3... 100%
rabbitmq v3.7.3 [Approved]
rabbitmq package files install completed. Performing other installation steps.
Downloading rabbitmq 
  from 'https://github.com/rabbitmq/rabbitmq-server/releases/download/v3.7.3/rabbitmq-server-3.7.3.exe'
Progress: 100% - Completed download of C:\Users\appveyor\AppData\Local\Temp\1\chocolatey\rabbitmq\3.7.3\rabbitmq-server-3.7.3.exe (11.04 MB).
Download of rabbitmq-server-3.7.3.exe (11.04 MB) completed.
Installing rabbitmq...
rabbitmq has been installed.
C:\Program Files\erl9.2\erts-9.2\bin\erlsrv: Service RabbitMQ enabled.
Enabling plugins on node rabbit@APPVYR-WIN:
rabbitmq_management
The following plugins have been configured:
  rabbitmq_management
  rabbitmq_management_agent
  rabbitmq_web_dispatch
Applying plugin configuration to rabbit@APPVYR-WIN...
The following plugins have been enabled:
  rabbitmq_management
  rabbitmq_management_agent
  rabbitmq_web_dispatch
enabled 3 plugins.
Offline change; changes will take effect at broker restart.
RabbitMQ service is already present - only updating service parameters
  rabbitmq may be able to be automatically uninstalled.
Environment Vars (like PATH) have changed. Close/reopen your shell to
 see the changes (or in powershell/cmd.exe just type `refreshenv`).
 The install of rabbitmq was successful.
  Software installed as 'EXE', install location is likely default.
Chocolatey installed 1/1 packages. 
 See the log for details (C:\ProgramData\chocolatey\logs\chocolatey.log).
Start-Sleep -s 2
go test -v ./...
=== RUN   TestNewBroker
--- PASS: TestNewBroker (0.00s)
=== RUN   ExampleMemoryQueue
--- PASS: ExampleMemoryQueue (0.00s)
PASS
ok  	gopkg.in/src-d/go-queue.v1	0.057s
=== RUN   TestAMQPSuite
=== RUN   TestAMQPSuite/TestConsume_empty
=== RUN   TestAMQPSuite/TestConsumersCanShareJobIteratorConcurrently
=== RUN   TestAMQPSuite/TestDelayed
=== RUN   TestAMQPSuite/TestJobIter_Next_empty
=== RUN   TestAMQPSuite/TestJob_Reject_no_requeue
=== RUN   TestAMQPSuite/TestJob_Reject_requeue
=== RUN   TestAMQPSuite/TestPublishAndConsume_immediate_ack
=== RUN   TestAMQPSuite/TestPublishDelayed_empty
=== RUN   TestAMQPSuite/TestPublishDelayed_nil
=== RUN   TestAMQPSuite/TestPublish_empty
=== RUN   TestAMQPSuite/TestPublish_nil
=== RUN   TestAMQPSuite/TestRetryQueue
=== RUN   TestAMQPSuite/TestTransaction
=== RUN   TestAMQPSuite/TestTransaction_Error
=== RUN   TestAMQPSuite/TestTransaction_not_supported
--- FAIL: TestAMQPSuite (10.60s)
    --- FAIL: TestAMQPSuite/TestConsume_empty (1.05s)
    	suite.go:39: 
    			Error Trace:	suite.go:39
    			            				suite.go:88
    			Error:      	Received unexpected error:
    			            	failed to connect to RabbitMQ: dial tcp 127.0.0.1:5672: connectex: No connection could be made because the target machine actively refused it.
    			            	
    			            	gopkg.in/src-d/go-queue.v1/amqp.New
    			            		c:/gopath/src/gopkg.in/src-d/go-queue.v1/amqp/amqp.go:78
    			            	gopkg.in/src-d/go-queue.v1/amqp.init.0.func1
    			            		c:/gopath/src/gopkg.in/src-d/go-queue.v1/amqp/amqp.go:27
    			            	gopkg.in/src-d/go-queue%2ev1.NewBroker
    			            		c:/gopath/src/gopkg.in/src-d/go-queue.v1/register.go:49
    			            	gopkg.in/src-d/go-queue.v1/test.(*QueueSuite).SetupTest
    			            		c:/gopath/src/gopkg.in/src-d/go-queue.v1/test/suite.go:38
    			            	github.com/stretchr/testify/suite.Run.func2
    			            		c:/gopath/src/github.com/stretchr/testify/suite/suite.go:88
    			            	testing.tRunner
    			            		C:/go/src/testing/testing.go:777
    			            	runtime.goexit
    			            		C:/go/src/runtime/asm_amd64.s:2361
    			Test:       	TestAMQPSuite/TestConsume_empty
    	suite.go:40: 
    			Error Trace:	suite.go:40
    			            				suite.go:88
    			Error:      	failed to connect to RabbitMQ: dial tcp 127.0.0.1:5672: connectex: No connection could be made because the target machine actively refused it.
    			Test:       	TestAMQPSuite/TestConsume_empty
    --- FAIL: TestAMQPSuite/TestConsumersCanShareJobIteratorConcurrently (1.20s)
    	suite.go:39: 
    			Error Trace:	suite.go:39
    			            				suite.go:88
    			Error:      	Received unexpected error:
    			            	failed to connect to RabbitMQ: dial tcp 127.0.0.1:5672: connectex: No connection could be made because the target machine actively refused it.
    			            	
    			            	gopkg.in/src-d/go-queue.v1/amqp.New
    			            		c:/gopath/src/gopkg.in/src-d/go-queue.v1/amqp/amqp.go:78
    			            	gopkg.in/src-d/go-queue.v1/amqp.init.0.func1
    			            		c:/gopath/src/gopkg.in/src-d/go-queue.v1/amqp/amqp.go:27
    			            	gopkg.in/src-d/go-queue%2ev1.NewBroker
    			            		c:/gopath/src/gopkg.in/src-d/go-queue.v1/register.go:49
    			            	gopkg.in/src-d/go-queue.v1/test.(*QueueSuite).SetupTest
    			            		c:/gopath/src/gopkg.in/src-d/go-queue.v1/test/suite.go:38
    			            	github.com/stretchr/testify/suite.Run.func2
    			            		c:/gopath/src/github.com/stretchr/testify/suite/suite.go:88
    			            	testing.tRunner
    			            		C:/go/src/testing/testing.go:777
    			            	runtime.goexit
    			            		C:/go/src/runtime/asm_amd64.s:2361
    			Test:       	TestAMQPSuite/TestConsumersCanShareJobIteratorConcurrently
    	suite.go:40: 
    			Error Trace:	suite.go:40
    			            				suite.go:88
    			Error:      	failed to connect to RabbitMQ: dial tcp 127.0.0.1:5672: connectex: No connection could be made because the target machine actively refused it.
    			Test:       	TestAMQPSuite/TestConsumersCanShareJobIteratorConcurrently
    --- FAIL: TestAMQPSuite/TestDelayed (1.00s)
    	suite.go:39: 
    			Error Trace:	suite.go:39
    			            				suite.go:88
    			Error:      	Received unexpected error:
    			            	failed to connect to RabbitMQ: dial tcp 127.0.0.1:5672: connectex: No connection could be made because the target machine actively refused it.
    			            	
    			            	gopkg.in/src-d/go-queue.v1/amqp.New
    			            		c:/gopath/src/gopkg.in/src-d/go-queue.v1/amqp/amqp.go:78
    			            	gopkg.in/src-d/go-queue.v1/amqp.init.0.func1
    			            		c:/gopath/src/gopkg.in/src-d/go-queue.v1/amqp/amqp.go:27
    			            	gopkg.in/src-d/go-queue%2ev1.NewBroker
    			            		c:/gopath/src/gopkg.in/src-d/go-queue.v1/register.go:49
    			            	gopkg.in/src-d/go-queue.v1/test.(*QueueSuite).SetupTest
    			            		c:/gopath/src/gopkg.in/src-d/go-queue.v1/test/suite.go:38
    			            	github.com/stretchr/testify/suite.Run.func2
    			            		c:/gopath/src/github.com/stretchr/testify/suite/suite.go:88
    			            	testing.tRunner
    			            		C:/go/src/testing/testing.go:777
    			            	runtime.goexit
    			            		C:/go/src/runtime/asm_amd64.s:2361
    			Test:       	TestAMQPSuite/TestDelayed
    	suite.go:40: 
    			Error Trace:	suite.go:40
    			            				suite.go:88
    			Error:      	failed to connect to RabbitMQ: dial tcp 127.0.0.1:5672: connectex: No connection could be made because the target machine actively refused it.
    			Test:       	TestAMQPSuite/TestDelayed
    --- FAIL: TestAMQPSuite/TestJobIter_Next_empty (1.00s)
    	suite.go:39: 
    			Error Trace:	suite.go:39
    			            				suite.go:88
    			Error:      	Received unexpected error:
    			            	failed to connect to RabbitMQ: dial tcp 127.0.0.1:5672: connectex: No connection could be made because the target machine actively refused it.
    			            	
    			            	gopkg.in/src-d/go-queue.v1/amqp.New
    			            		c:/gopath/src/gopkg.in/src-d/go-queue.v1/amqp/amqp.go:78
    			            	gopkg.in/src-d/go-queue.v1/amqp.init.0.func1
    			            		c:/gopath/src/gopkg.in/src-d/go-queue.v1/amqp/amqp.go:27
    			            	gopkg.in/src-d/go-queue%2ev1.NewBroker
    			            		c:/gopath/src/gopkg.in/src-d/go-queue.v1/register.go:49
    			            	gopkg.in/src-d/go-queue.v1/test.(*QueueSuite).SetupTest
    			            		c:/gopath/src/gopkg.in/src-d/go-queue.v1/test/suite.go:38
    			            	github.com/stretchr/testify/suite.Run.func2
    			            		c:/gopath/src/github.com/stretchr/testify/suite/suite.go:88
    			            	testing.tRunner
    			            		C:/go/src/testing/testing.go:777
    			            	runtime.goexit
    			            		C:/go/src/runtime/asm_amd64.s:2361
    			Test:       	TestAMQPSuite/TestJobIter_Next_empty
    	suite.go:40: 
    			Error Trace:	suite.go:40
    			            				suite.go:88
    			Error:      	failed to connect to RabbitMQ: dial tcp 127.0.0.1:5672: connectex: No connection could be made because the target machine actively refused it.
    			Test:       	TestAMQPSuite/TestJobIter_Next_empty
    --- FAIL: TestAMQPSuite/TestJob_Reject_no_requeue (1.03s)
    	suite.go:39: 
    			Error Trace:	suite.go:39
    			            				suite.go:88
    			Error:      	Received unexpected error:
    			            	failed to connect to RabbitMQ: dial tcp 127.0.0.1:5672: connectex: No connection could be made because the target machine actively refused it.
    			            	
    			            	gopkg.in/src-d/go-queue.v1/amqp.New
    			            		c:/gopath/src/gopkg.in/src-d/go-queue.v1/amqp/amqp.go:78
    			            	gopkg.in/src-d/go-queue.v1/amqp.init.0.func1
    			            		c:/gopath/src/gopkg.in/src-d/go-queue.v1/amqp/amqp.go:27
    			            	gopkg.in/src-d/go-queue%2ev1.NewBroker
    			            		c:/gopath/src/gopkg.in/src-d/go-queue.v1/register.go:49
    			            	gopkg.in/src-d/go-queue.v1/test.(*QueueSuite).SetupTest
    			            		c:/gopath/src/gopkg.in/src-d/go-queue.v1/test/suite.go:38
    			            	github.com/stretchr/testify/suite.Run.func2
    			            		c:/gopath/src/github.com/stretchr/testify/suite/suite.go:88
    			            	testing.tRunner
    			            		C:/go/src/testing/testing.go:777
    			            	runtime.goexit
    			            		C:/go/src/runtime/asm_amd64.s:2361
    			Test:       	TestAMQPSuite/TestJob_Reject_no_requeue
    	suite.go:40: 
    			Error Trace:	suite.go:40
    			            				suite.go:88
    			Error:      	failed to connect to RabbitMQ: dial tcp 127.0.0.1:5672: connectex: No connection could be made because the target machine actively refused it.
    			Test:       	TestAMQPSuite/TestJob_Reject_no_requeue
    --- FAIL: TestAMQPSuite/TestJob_Reject_requeue (1.02s)
    	suite.go:39: 
    			Error Trace:	suite.go:39
    			            				suite.go:88
    			Error:      	Received unexpected error:
    			            	failed to connect to RabbitMQ: dial tcp 127.0.0.1:5672: connectex: No connection could be made because the target machine actively refused it.
    			            	
    			            	gopkg.in/src-d/go-queue.v1/amqp.New
    			            		c:/gopath/src/gopkg.in/src-d/go-queue.v1/amqp/amqp.go:78
    			            	gopkg.in/src-d/go-queue.v1/amqp.init.0.func1
    			            		c:/gopath/src/gopkg.in/src-d/go-queue.v1/amqp/amqp.go:27
    			            	gopkg.in/src-d/go-queue%2ev1.NewBroker
    			            		c:/gopath/src/gopkg.in/src-d/go-queue.v1/register.go:49
    			            	gopkg.in/src-d/go-queue.v1/test.(*QueueSuite).SetupTest
    			            		c:/gopath/src/gopkg.in/src-d/go-queue.v1/test/suite.go:38
    			            	github.com/stretchr/testify/suite.Run.func2
    			            		c:/gopath/src/github.com/stretchr/testify/suite/suite.go:88
    			            	testing.tRunner
    			            		C:/go/src/testing/testing.go:777
    			            	runtime.goexit
    			            		C:/go/src/runtime/asm_amd64.s:2361
    			Test:       	TestAMQPSuite/TestJob_Reject_requeue
    	suite.go:40: 
    			Error Trace:	suite.go:40
    			            				suite.go:88
    			Error:      	failed to connect to RabbitMQ: dial tcp 127.0.0.1:5672: connectex: No connection could be made because the target machine actively refused it.
    			Test:       	TestAMQPSuite/TestJob_Reject_requeue
    --- FAIL: TestAMQPSuite/TestPublishAndConsume_immediate_ack (1.01s)
    	suite.go:39: 
    			Error Trace:	suite.go:39
    			            				suite.go:88
    			Error:      	Received unexpected error:
    			            	failed to connect to RabbitMQ: dial tcp 127.0.0.1:5672: connectex: No connection could be made because the target machine actively refused it.
    			            	
    			            	gopkg.in/src-d/go-queue.v1/amqp.New
    			            		c:/gopath/src/gopkg.in/src-d/go-queue.v1/amqp/amqp.go:78
    			            	gopkg.in/src-d/go-queue.v1/amqp.init.0.func1
    			            		c:/gopath/src/gopkg.in/src-d/go-queue.v1/amqp/amqp.go:27
    			            	gopkg.in/src-d/go-queue%2ev1.NewBroker
    			            		c:/gopath/src/gopkg.in/src-d/go-queue.v1/register.go:49
    			            	gopkg.in/src-d/go-queue.v1/test.(*QueueSuite).SetupTest
    			            		c:/gopath/src/gopkg.in/src-d/go-queue.v1/test/suite.go:38
    			            	github.com/stretchr/testify/suite.Run.func2
    			            		c:/gopath/src/github.com/stretchr/testify/suite/suite.go:88
    			            	testing.tRunner
    			            		C:/go/src/testing/testing.go:777
    			            	runtime.goexit
    			            		C:/go/src/runtime/asm_amd64.s:2361
    			Test:       	TestAMQPSuite/TestPublishAndConsume_immediate_ack
    	suite.go:40: 
    			Error Trace:	suite.go:40
    			            				suite.go:88
    			Error:      	failed to connect to RabbitMQ: dial tcp 127.0.0.1:5672: connectex: No connection could be made because the target machine actively refused it.
    			Test:       	TestAMQPSuite/TestPublishAndConsume_immediate_ack
    --- FAIL: TestAMQPSuite/TestPublishDelayed_empty (1.02s)
    	suite.go:39: 
    			Error Trace:	suite.go:39
    			            				suite.go:88
    			Error:      	Received unexpected error:
    			            	failed to connect to RabbitMQ: dial tcp 127.0.0.1:5672: connectex: No connection could be made because the target machine actively refused it.
    			            	
    			            	gopkg.in/src-d/go-queue.v1/amqp.New
    			            		c:/gopath/src/gopkg.in/src-d/go-queue.v1/amqp/amqp.go:78
    			            	gopkg.in/src-d/go-queue.v1/amqp.init.0.func1
    			            		c:/gopath/src/gopkg.in/src-d/go-queue.v1/amqp/amqp.go:27
    			            	gopkg.in/src-d/go-queue%2ev1.NewBroker
    			            		c:/gopath/src/gopkg.in/src-d/go-queue.v1/register.go:49
    			            	gopkg.in/src-d/go-queue.v1/test.(*QueueSuite).SetupTest
    			            		c:/gopath/src/gopkg.in/src-d/go-queue.v1/test/suite.go:38
    			            	github.com/stretchr/testify/suite.Run.func2
    			            		c:/gopath/src/github.com/stretchr/testify/suite/suite.go:88
    			            	testing.tRunner
    			            		C:/go/src/testing/testing.go:777
    			            	runtime.goexit
    			            		C:/go/src/runtime/asm_amd64.s:2361
    			Test:       	TestAMQPSuite/TestPublishDelayed_empty
    	suite.go:40: 
    			Error Trace:	suite.go:40
    			            				suite.go:88
    			Error:      	failed to connect to RabbitMQ: dial tcp 127.0.0.1:5672: connectex: No connection could be made because the target machine actively refused it.
    			Test:       	TestAMQPSuite/TestPublishDelayed_empty
    --- FAIL: TestAMQPSuite/TestPublishDelayed_nil (1.02s)
    	suite.go:39: 
    			Error Trace:	suite.go:39
    			            				suite.go:88
    			Error:      	Received unexpected error:
    			            	failed to connect to RabbitMQ: dial tcp 127.0.0.1:5672: connectex: No connection could be made because the target machine actively refused it.
    			            	
    			            	gopkg.in/src-d/go-queue.v1/amqp.New
    			            		c:/gopath/src/gopkg.in/src-d/go-queue.v1/amqp/amqp.go:78
    			            	gopkg.in/src-d/go-queue.v1/amqp.init.0.func1
    			            		c:/gopath/src/gopkg.in/src-d/go-queue.v1/amqp/amqp.go:27
    			            	gopkg.in/src-d/go-queue%2ev1.NewBroker
    			            		c:/gopath/src/gopkg.in/src-d/go-queue.v1/register.go:49
    			            	gopkg.in/src-d/go-queue.v1/test.(*QueueSuite).SetupTest
    			            		c:/gopath/src/gopkg.in/src-d/go-queue.v1/test/suite.go:38
    			            	github.com/stretchr/testify/suite.Run.func2
    			            		c:/gopath/src/github.com/stretchr/testify/suite/suite.go:88
    			            	testing.tRunner
    			            		C:/go/src/testing/testing.go:777
    			            	runtime.goexit
    			            		C:/go/src/runtime/asm_amd64.s:2361
    			Test:       	TestAMQPSuite/TestPublishDelayed_nil
    	suite.go:40: 
    			Error Trace:	suite.go:40
    			            				suite.go:88
    			Error:      	failed to connect to RabbitMQ: dial tcp 127.0.0.1:5672: connectex: No connection could be made because the target machine actively refused it.
    			Test:       	TestAMQPSuite/TestPublishDelayed_nil
    --- PASS: TestAMQPSuite/TestPublish_empty (0.53s)
    --- PASS: TestAMQPSuite/TestPublish_nil (0.09s)
    --- PASS: TestAMQPSuite/TestRetryQueue (0.53s)
    --- PASS: TestAMQPSuite/TestTransaction (0.02s)
    --- PASS: TestAMQPSuite/TestTransaction_Error (0.07s)
    --- SKIP: TestAMQPSuite/TestTransaction_not_supported (0.00s)
    	suite.go:438: transactions supported
=== RUN   TestDefaultConfig
--- PASS: TestDefaultConfig (0.00s)
=== RUN   TestNewAMQPBroker_bad_url
--- PASS: TestNewAMQPBroker_bad_url (0.00s)
=== RUN   TestAMQPPriorities
--- PASS: TestAMQPPriorities (0.33s)
=== RUN   TestAMQPHeaders
=== RUN   TestAMQPHeaders/with_x-retries_and_x-error-type_headers
=== RUN   TestAMQPHeaders/with_x-retries_header
=== RUN   TestAMQPHeaders/with_x-error-type_headers
=== RUN   TestAMQPHeaders/with_no_headers
--- PASS: TestAMQPHeaders (0.05s)
    --- PASS: TestAMQPHeaders/with_x-retries_and_x-error-type_headers (0.00s)
    --- PASS: TestAMQPHeaders/with_x-retries_header (0.00s)
    --- PASS: TestAMQPHeaders/with_x-error-type_headers (0.00s)
    --- PASS: TestAMQPHeaders/with_no_headers (0.00s)
=== RUN   TestAMQPRepublishBuried
--- PASS: TestAMQPRepublishBuried (4.71s)
FAIL
FAIL	gopkg.in/src-d/go-queue.v1/amqp	16.470s
=== RUN   TestMemorySuite
=== RUN   TestMemorySuite/TestConsume_empty
=== RUN   TestMemorySuite/TestConsumersCanShareJobIteratorConcurrently
=== RUN   TestMemorySuite/TestDelayed
=== RUN   TestMemorySuite/TestIntegration
=== RUN   TestMemorySuite/TestJobIter_Next_empty
=== RUN   TestMemorySuite/TestJob_Reject_no_requeue
=== RUN   TestMemorySuite/TestJob_Reject_requeue
=== RUN   TestMemorySuite/TestPublishAndConsume_immediate_ack
=== RUN   TestMemorySuite/TestPublishDelayed_empty
=== RUN   TestMemorySuite/TestPublishDelayed_nil
=== RUN   TestMemorySuite/TestPublish_empty
=== RUN   TestMemorySuite/TestPublish_nil
=== RUN   TestMemorySuite/TestRetryQueue
=== RUN   TestMemorySuite/TestTransaction
=== RUN   TestMemorySuite/TestTransaction_Error
=== RUN   TestMemorySuite/TestTransaction_not_supported
--- PASS: TestMemorySuite (1.33s)
    --- PASS: TestMemorySuite/TestConsume_empty (0.00s)
    --- PASS: TestMemorySuite/TestConsumersCanShareJobIteratorConcurrently (0.00s)
    --- PASS: TestMemorySuite/TestDelayed (1.25s)
    --- PASS: TestMemorySuite/TestIntegration (0.06s)
    --- PASS: TestMemorySuite/TestJobIter_Next_empty (0.00s)
    --- PASS: TestMemorySuite/TestJob_Reject_no_requeue (0.00s)
    --- PASS: TestMemorySuite/TestJob_Reject_requeue (0.00s)
    --- PASS: TestMemorySuite/TestPublishAndConsume_immediate_ack (0.00s)
    --- PASS: TestMemorySuite/TestPublishDelayed_empty (0.00s)
    --- PASS: TestMemorySuite/TestPublishDelayed_nil (0.00s)
    --- PASS: TestMemorySuite/TestPublish_empty (0.00s)
    --- PASS: TestMemorySuite/TestPublish_nil (0.00s)
    --- PASS: TestMemorySuite/TestRetryQueue (0.00s)
    --- PASS: TestMemorySuite/TestTransaction (0.00s)
    --- PASS: TestMemorySuite/TestTransaction_Error (0.00s)
    --- SKIP: TestMemorySuite/TestTransaction_not_supported (0.00s)
    	suite.go:438: transactions supported
PASS
ok  	gopkg.in/src-d/go-queue.v1/memory	1.879s
?   	gopkg.in/src-d/go-queue.v1/test	[no test files]
Command exited with code 1
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -1,9 +1,8 @@
 language: go
 
 go:
-  - 1.9.x
+  - 1.11.x
   - 1.10.x
-  - tip
 
 go_import_path: gopkg.in/src-d/go-queue.v1
 

--- a/Makefile
+++ b/Makefile
@@ -3,12 +3,10 @@ PROJECT = go-queue
 COMMANDS =
 
 # Including ci Makefile
-MAKEFILE = Makefile.main
-CI_REPOSITORY = https://github.com/src-d/ci.git
-CI_FOLDER = .ci
-
+CI_REPOSITORY ?= https://github.com/src-d/ci.git
+CI_BRANCH ?= v1
+CI_PATH ?= .ci
+MAKEFILE := $(CI_PATH)/Makefile.main
 $(MAKEFILE):
-	@git clone --quiet $(CI_REPOSITORY) $(CI_FOLDER); \
-	cp $(CI_FOLDER)/$(MAKEFILE) .;
-
+	git clone --quiet --depth 1 -b $(CI_BRANCH) $(CI_REPOSITORY) $(CI_PATH);
 -include $(MAKEFILE)

--- a/amqp/amqp_test.go
+++ b/amqp/amqp_test.go
@@ -3,6 +3,7 @@ package queue
 import (
 	"context"
 	"fmt"
+	"os"
 	"os/exec"
 	"testing"
 	"time"
@@ -15,11 +16,12 @@ import (
 	"github.com/stretchr/testify/suite"
 )
 
-// RabbitMQ tests require running docker.
+// RabbitMQ reconnect tests require running docker.
 // If `docker ps` command returned an error we skip some of the tests.
 var (
 	dockerIsRunning bool
 	dockerCmdOutput string
+	inAppVeyor      bool
 )
 
 func init() {
@@ -27,6 +29,7 @@ func init() {
 	b, err := cmd.CombinedOutput()
 
 	dockerCmdOutput, dockerIsRunning = string(b), (err == nil)
+	inAppVeyor = os.Getenv("APPVEYOR") == "True"
 }
 
 func TestAMQPSuite(t *testing.T) {
@@ -226,7 +229,7 @@ func TestAMQPRepublishBuried(t *testing.T) {
 }
 
 func TestReconnect(t *testing.T) {
-	if !dockerIsRunning {
+	if inAppVeyor || !dockerIsRunning {
 		t.Skip()
 	}
 

--- a/appveyor.yml
+++ b/appveyor.yml
@@ -16,7 +16,7 @@ install:
   - mingw32-make ci-install
 
 test_script:
-  - mingw32-make ci-script
+  - mingw32-make test
 
 build: off
 deploy: off

--- a/appveyor.yml
+++ b/appveyor.yml
@@ -1,20 +1,22 @@
 version: "{build}"
+image: Visual Studio 2017
 platform: x64
 
 clone_folder: c:\gopath\src\gopkg.in\src-d\go-queue.v1
 
 environment:
   GOPATH: c:\gopath
+  RABBITMQ_VERSION: any
 
-before_test:
-  - set PATH=C:\Program Files\erl9.2\bin;%PATH%
-  - choco install rabbitmq --ignoredependencies -y
-  - ps: Start-Sleep -s 2
+cache:
+  - C:\Users\appveyor\AppData\Local\Temp\chocolatey\ -> appveyor.yml
 
-build_script:
-  - set PATH=%GOPATH%\bin;c:\go\bin;%PATH%
-  - go version
-  - go get -v -t ./...
+install:
+  - set PATH=%GOPATH%\bin;C:\go\bin;C:\mingw-w64\x86_64-7.2.0-posix-seh-rt_v5-rev1\mingw64\bin;C:\msys64\usr\bin;%PATH%;"C:\Program Files\Git\mingw64\bin"
+  - mingw32-make ci-install
 
 test_script:
-  - go test -v ./...
+  - mingw32-make ci-script
+
+build: off
+deploy: off

